selenium_tests/course_info/course_info_details_noneditable_course_tests.py

```diff
@@ -0,0 +1,58 @@
+from selenium_tests.course_info.course_info_base_test_case \
+    import CourseInfoBaseTestCase
+from selenium_tests.course_info.page_objects.course_info_detail_page_object \
+    import  Locators
+
+
+class CourseInfoDetailsNonEditTests(CourseInfoBaseTestCase):
+
+    def setUp(self):
+        super(CourseInfoDetailsNonEditTests, self).setUp()
+        # Load a non-ILE course to validate non editable courses
+        self._load_test_course()
+        self.assertTrue(self.detail_page.is_loaded())
+
+
+    def test_regular_course_not_editable(self):
+        """
+        verify non-SB/ILE courses cannot be edited
+        """
+        # Assert that the Save and Reset button are not present
+        self.assertEquals(self.detail_page.is_locator_element_present
```

Comment:
Typical page object design is such that the PO's locators and the page's internals are not exposed to the test. Instead, the PO exposes services like `is_course_editable()`, which then use the appropriate Locators internally to return a true or false. The test should not have to know about what the locators are.